apps/dashboard/src/routes/dashboard/+server.ts

```diff
@@ -0,0 +1,20 @@
+import { json, error } from '@sveltejs/kit';
+
+export const POST = async ({ request, locals: { supabase, getSession } }) => {
```

Comment:
As noted in my other comment, lets keep API calls under /api/ and if we can avoid having them tied to specific UI.

So if this POST call creates links, I would have it as: `POST /api/links`.

From a customer perspective they are just creating "links".